File: py/Length of Last Word.py
class Solution:
    # @param s, a string
    # @return an integer
    def lengthOfLastWord(self, s):
        l = s.split(' ')
            
        while len(l) != 0:
            tmp = l.pop()
            if tmp.isalpha():
                return len(tmp)
            
        return 0

File: cpp/char_problem/lengthofLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(const char *s) {
        
        int len = 0;
        const char* p = s;
        
        while(p != NULL && *p != '\0')
        {
            p += 1;
            len += 1;
        }
        
        if(len == 0)
            return 0;
            
        int iret = 0;
        
        for(int i = len-1;i >= 0;i--)
        {
            if(s[i] == ' ')
                continue;
            else
            {
                while(i >= 0 && s[i] != ' ')
                {
                    iret += 1;
                    i -= 1;
                }
                break;
            }
        }
        
        return iret;
    }
};